src/views/preview/project-view.jsx

```diff
@@ -32,6 +33,8 @@ const frameless = require('../../lib/frameless');
 const GUI = require('scratch-gui');
 const IntlGUI = injectIntl(GUI.default);
 
+const USE_SCRATCH3_REGISTRATION = false;
```

Comment:
What would it take to use the same flag to enable this for both navigation and the editor?

Reply:
Done!